lib/commands.js

```diff
@@ -597,6 +601,15 @@ var operations = module.exports.operations = {
       context.overrides.kms,
       finished
     );
+  },
+
+  mergeMetadata: function(context) {
+    if (!context.overrides.metadata) return context.next();
+    context.newTemplate.Metadata = context.newTemplate.Metadata || {};
+    for (var k in context.overrides.metadata) {
+      context.newTemplate.Metadata[k] = context.overrides.metadata[k];
```

Comment:
Any fear of overwriting? Cloudfriend's approach is to fail hard on overwrites -- I feel like that has the potential to  avoid some confusing situations somewhere down that road.